lib/text/ui_text_displayer.js

```diff
@@ -504,6 +508,19 @@ shaka.text.UITextDisplayer = class {
       regionElement.style.left = region.viewportAnchorX -
         region.regionAnchorX * region.width / 100 + viewportAnchorUnit;
     }
+    if (region.heightUnits !== pixelUnit &&
```

Comment:
I don't get why you are checking, for these things, `!== pixelUnit`, instead of checking `=== percentageUnit`.
That would suggest that this code works for both percentages and lines.
This code assumes that the max for width and height are both 100, which is true for percentages, but lines max out at 16 or 32 depending on the aspect ratio.

Reply:
Internally we convert the lines to % that's why I don't check percentages